frame/support/procedural/src/construct_runtime/expand/event.rs

```diff
@@ -30,7 +30,7 @@ pub fn expand_outer_event(
 	let mut query_event_part_macros = Vec::new();
 
 	for pallet_decl in pallet_decls {
-		if let Some(pallet_entry) = pallet_decl.find_part("Event") {
```

Comment:
Sorry but I don't think this is necessary. The pallet part has always been called `Event`, and from context we know for sure that it's referring to the pallet event type, and not the runtime event type, so please revert this.

Reply:
Ok, so just to make sure I understand this correctly, the `Event` enum inside a pallet shouldn't be renamed to `PalletEvent`, but when accessing it from another pallet it should be accessed with `PalletEvent`?

Reply:
No, I think you misunderstood.

We _do not_ need to rename the `Event` pallet part. This here is where an `Event` pallet part is used:

```rust
construct_runtime!(
	pub enum Runtime where
		Block = Block,
		NodeBlock = Block,
		UncheckedExtrinsic = UncheckedExtrinsic,
	{
		System: frame_system::{Pallet, Call, Storage, Config, Event<T>},
	}
);
```

We _do_ need to rename the `Event` pallet enum to `PalletEvent`, i.e. whenever you see something like `#path::Event::<#runtime, #path::#inst>` or something similar to this type, you'll have to rename the `Event` there to `PalletEvent`.

In your latest push, you have completely removed all `PalletEvent`s. We need to add them back.